edns.go

```diff
@@ -78,7 +78,10 @@ func (rr *OPT) String() string {
 	if rr.Do() {
 		s += "flags: do; "
 	} else {
-		s += "flags: ; "
+		s += "flags:; "
+	}
+	if rr.Hdr.Ttl&0x7FFF != 0 {
+		s += "MBZ: " + fmt.Sprintf("0x%04x", rr.Hdr.Ttl&0x7FFF) + ", "
```

Comment:
Put the prefix and suffix into the `fmt.Sprintf` call. No point in adding an extra allocation where it’s not needed.

Reply:
Done